### DISCUSSION TITLE
成大和廣達電腦簽署合作備忘錄 成立AI研究中心

### DISCUSSION THREAD

**Discussion by kaoyihua27** [1101-GS3073-C/Python-Community-GS3073-C#173] [General]
好希望中央也能有一樣的機會，讓我們更多發展~
https://udn.com/news/story/6928/5996549

**Comment by pocper1** [comment]
成大好好！ 每年有1000萬元的設備補助

**Comment by bonny123865** [comment]
這個企業與學校的產學合作除了能為台灣科技帶來更好的發展，說不定可以為南部帶來商機！

**Comment by chia-kai-chang** [comment]
錢能解決的都是小事。買器材就能解決的事，對廣達來說是九牛一毛，廣達在這個合作中一定也得的學校/學生的軟實力，這才是廣達想要的。
大家也不要洩氣，我們在通識中心的【人工智慧跨域應用學分學程】一步一步學習，機會不輸別人。 😂

**Comment by briankuo123** [comment]
聽說中央大學拿的錢很少，我認為除非我們自己先有達到一個創舉讓政府或企業有意願來投資我們，不然應該很難有機會。

**Comment by wh20502** [comment]
不確定會不會綁約欸，有些企業計劃會要畢業後必須服務幾年，但是畢業不會失業好像也挺不錯的。

**Comment by Shiba010** [comment]
中央也要加油！希望在未來也能爭取到類似的機會！
